qutebrowser/browser/commands.py

```diff
@@ -1147,6 +1170,21 @@ def bookmark_load(self, url, tab=False, bg=False, window=False):
             raise cmdexc.CommandError(e)
         self._open(url, tab, bg, window)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window',
+                       maxsplit=0)
+    @cmdutils.argument('url', completion=usertypes.Completion.bookmark_by_url)
+    def bookmark_del(self, url=None):
+        """Delete a bookmark.
+
+        Args:
+            url: The url of the bookmark to delete. If None, use the
+                 current page's url.
+        """
+        if url is None:
+            url = self._current_url().toString(QUrl.RemovePassword
+                                             | QUrl.FullyEncoded)
+        objreg.get('bookmark-manager').bookmark_del(url)
```

Comment:
Same as with quickmarks - probably better to remove `bookmark_del` on the `BookmarkManager`.